src/app/common/transactions/bitcoin/utils.ts

```diff
@@ -80,22 +80,22 @@ export function filterUneconomicalUtxos({
   return filteredUtxos;
 }
 
-export function getSizeInfo(payload: {
-  inputLength: number;
-  outputLength: number;
+export function getBitcoinTxSizeEstimation(payload: {
+  inputCount: number;
+  outputCount: number;
   recipient: string;
 }) {
-  const { inputLength, recipient, outputLength } = payload;
+  const { inputCount, recipient, outputCount } = payload;
   const addressInfo = validate(recipient) ? getAddressInfo(recipient) : null;
   const outputAddressTypeWithFallback = addressInfo ? addressInfo.type : 'p2wpkh';
 
   const txSizer = new BtcSizeFeeEstimator();
   const sizeInfo = txSizer.calcTxSize({
     // Only p2wpkh is supported by the wallet
     input_script: 'p2wpkh',
-    input_count: inputLength,
+    input_count: inputCount,
     // From the address of the recipient, we infer the output type
-    [outputAddressTypeWithFallback + '_output_count']: outputLength,
+    [outputAddressTypeWithFallback + '_output_count']: outputCount,
```

Comment:
as I understand we can skip adding `_outputs_counts` for different output types? 🤔
like it is now in `getSizeInfoMultipleRecipients`

Reply:
Not entirely sure what you mean by this @alter-eggo ?

Reply:
You do this as well in the multiple recipients code ` acc[type + '_output_count'] = count;`